core/challenges.go

```diff
@@ -40,7 +40,9 @@ func HTTPChallenge01(accountKey *jose.JsonWebKey) Challenge {
 
 // TLSSNIChallenge01 constructs a random tls-sni-00 challenge
 func TLSSNIChallenge01(accountKey *jose.JsonWebKey) Challenge {
-	return newChallenge(ChallengeTypeTLSSNI01, accountKey)
+	chall := newChallenge(ChallengeTypeTLSSNI01, accountKey)
+	chall.N = 25
```

Comment:
Let's not hard-wire this.  Instead, have this method take a second parameter, and have the policy authority set `N`.  If you want to hard-code something put it there.